### DISCUSSION TITLE
Install `bbin` together with `bb`

### DISCUSSION THREAD

**Discussion by filipesilva** [babashka/babashka#1774] [Ideas]
**Is your feature request related to a problem? Please describe.**
I'm working on a couple of CLI tools and would like to give users simple pre-requisite instructions. For clojure projects, the pre-requisites start with java and clojure. I think babashka is the nicest way of doing CLI tools in clojure-land, so that's another. But the nicest way to install babashka CLIs is bbin, so that is a fourth pre-requisite.

**Describe the solution you'd like**
I think it would be cool if `bbin` was installed alongside `bb` when using the `bb` installers, since it's effectively the `bb` global package manager. Since such a high % of Clojure users also have `bb`, they would automatically have `bbin`, and it would be easier for bb tools creators to distribute their projects.

**Describe alternatives you've considered**
Directing users to install `bbin` directly is fine too. It's not a dealbreaker by any means.

**Additional context**
Babashka is awesome!

**Comment by borkdude** [comment]
Initially the plan was to have a built-in command called `bb install`. But this seemed to risky so we decided to do this as a separate tool which was called bbin. One day the separate tool might become bb install though.
